boards/arm/mimxrt685_evk/board.cmake

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2020, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+board_set_debugger_ifnset(jlink)
+board_set_flasher_ifnset(jlink)
```

Comment:
Remove these lines. Including `jlink.board.cmake` below takes care of this

Reply:
Done